remoto/tests/test_log.py

```diff
@@ -36,6 +36,30 @@ def test_strip_new_lines(self):
         message = conn.logger.error.call_args[0][0]
         assert message == 'an error message'
 
+    def test_strip_new_lines_extra_1(self):
```

Comment:
Correct！Meaningful name is better to the code maintenance.The code will be updated.